mobile-save-for-later-user-deletion/template.yml

```diff
@@ -0,0 +1,84 @@
+# This AWS SAM template has been generated from your function's configuration. If
```

Comment:
Is it the cloudformation for mobile-save-for-later?  Why do we need this one when there is "conf/cfn.yaml"?